federation-gateway-bundle/feg-eap-aka-operator/tox.ini

```
@@ -0,0 +1,87 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, static, unit
+
+[vars]
+src_path = {toxinidir}/src/
+unit_test_path = {toxinidir}/tests/unit/
+integration_test_path = {toxinidir}/tests/integration/
+all_path = {[vars]src_path} {[vars]unit_test_path} {[vars]integration_test_path}
+
+[testenv]
+deps = 
+    pytest
+    pytest-operator
+setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+    PYTHONBREAKPOINT=ipdb.set_trace
+passenv =
+    HTTP_PROXY
+    HTTPS_PROXY
+    NO_PROXY
+    PYTHONPATH
+    HOME
+    CHARM_BUILD_DIR
+    MODEL_SETTINGS
+
+[testenv:fmt]
+description = Apply coding style standards to code
+deps =
+    black
+    isort
+commands =
+    isort {[vars]all_path}
+    black {[vars]all_path}
+
+[testenv:lint]
+description = Check code against coding style standards
+deps =
+    black
+    flake8
+    flake8-docstrings
+    flake8-copyright
+    flake8-builtins
+    pyproject-flake8
+    pep8-naming
+    isort
+commands =
+    pflake8 {[vars]all_path}
+    isort --check-only --diff {[vars]all_path}
+    black --check --diff {[vars]all_path}
+
+[testenv:static]
+description = Run static analysis checks
+deps =
+    -r{toxinidir}/requirements.txt
+    mypy
+    types-PyYAML
+    pytest
+    pytest-operator
+    juju
+    types-setuptools
+    types-toml
+commands =
```

This will fix the static analysis:
```suggestion
    types-toml
setenv =
    PYTHONPATH = ""
commands =
```

---

Fixed